kubernetes/moosefs-csi-plugin.yaml

```diff
@@ -27,8 +28,8 @@ spec:
             capabilities:
               add: ["SYS_ADMIN"]
             allowPrivilegeEscalation: true
-          image: kunde21/moosefs-csi
-          args :
+          image: kunde21/moosefs-csi:v3.0.116-v0.1.1
```

Comment:
@Kunde21 I was shooting for something that just works out of the box. Coincidentally, I think installing via the kustomize directory today would give you 3.0.115 instead of the 3.0.116 tag here.

Unrelated, but re: this part of the change

> Modifies CSI containers to use official (+multi-platform) sig-storage images.

This actually broke things significantly. It's been a while but I think I was seeing connections to volumes drop and then never re-mount. It looks like #2 might have resolved that via your custom CSI images, but I couldn't find the source for those. Do you still have that handy? I might be interested in helping with snapshot support.